scene/gui/progress_bar.h

```diff
@@ -43,11 +43,27 @@ class ProgressBar : public Range {
 	static void _bind_methods();
 
 public:
+	enum FillMode {
+		FILL_LEFT_TO_RIGHT = 0,
```

Comment:
Is the `= 0` necessary here? Enums start at 0 by default. I think the value should only be specified if it actually overrides the value (e.g. to allow adding values in between without breaking compatibility).

Reply:
@Calinou No, it's not. I copied this enum from TextureProgressBar, and it so happens that it's the case there.